packages/rum/test/trackEventCounts.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { ErrorMessage, objectValues } from '@datadog/browser-core'
+import { LifeCycle, LifeCycleEventType } from '../src/lifeCycle'
+import { EventCounts, trackEventCounts } from '../src/trackEventCounts'
+import { UserAction } from '../src/userActionCollection'
+
+describe('trackEventCounts', () => {
+  it('tracks errors', () => {
+    const lifeCycle = new LifeCycle()
+    const { eventCounts } = trackEventCounts(lifeCycle)
+    // tslint:disable-next-line no-object-literal-type-assertion
+    lifeCycle.notify(LifeCycleEventType.ERROR_COLLECTED, {} as ErrorMessage)
```

Comment:
instead of having this tslint comment, what about having for each test something like:
```
const event: ErrorMessage = {}
lifeCycle.notify(LifeCycleEventType.ERROR_COLLECTED, event)
```
it would remove the noise and add some consistency between the tests